package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "snoozetabs",
+  "name": "Snooze Tabs",
```

Comment:
I don't think package.json files can have spaces in the `name` property.
I think we may have to set the `title` property: https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/package_json